src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.Netcore.csproj

```diff
@@ -27,7 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Dns" Version="2.2.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.1.0-preview" />
```

Comment:
Can you delete local feed packages for this? Also, I am assuming you are generating this from the latest swagger branch?

Reply:
Rebased on the target branch.
No, this PR just keeps the version that is already in LocalFeed